doc/source/security/encrypting-tables.rst

```diff
@@ -27,13 +23,13 @@ An example of the ``CREATE TABLE`` statement:
 
 .. code-block:: mysql
 
-   mysql> CREATE TABLE myexample (id INT, mytext varchar(255)) ENCRYPTION='Y';
+   mysql> CREATE TABLE sample (id INT, mytext varchar(255)) ENCRYPTION='Y';
 
 An example of an ``ALTER TABLE`` statement.
 
-.. code-block:: MySQL
+.. code-block:: mysql
```

Comment:
any difference when code-block is mysql vs text ?

Reply:
code-block with MySQL adds syntax coding for the code sample.